app/controllers/NiLetterController.scala

```diff
@@ -39,24 +41,31 @@ class NiLetterController @Inject() (
   errorRenderer: ErrorRenderer,
   printNiNumberView: PrintNationalInsuranceNumberView,
   pdfWrapperView: NiLetterPDfWrapperView,
-  niLetterView: NiLetterView
+  niLetterView: NiLetterView,
+  featureFlagService: FeatureFlagService
 )(implicit configDecorator: ConfigDecorator, val ec: ExecutionContext)
     extends PertaxBaseController(cc) {
 
   def printNationalInsuranceNumber: Action[AnyContent] =
```

Comment:
I would not have the change in here but there:
https://github.com/hmrc/pertax-frontend/blob/6b2065329cce196e30b549f779a19e6655653346/app/viewmodels/PersonalDetailsViewModel.scala#L159